package.json

```diff
@@ -52,5 +52,8 @@
   "engines": {
     "node": ">=0.10.0"
   },
-  "engineStrict": true
+  "engineStrict": true,
+  "dependencies": {
+    "moment": "^2.19.2"
+  }
```

Comment:
Do we need this dependency when we are using the js/lib file?

Reply:
I don't think so, but might be smart to keep in there for when we move to hoot 2.0.